includes/payment-methods/Configs/Definitions/LinkDefinition.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * Link Payment Method Definition
+ *
+ * @package WCPay\PaymentMethods\Configs\Definitions
+ */
+
+namespace WCPay\PaymentMethods\Configs\Definitions;
+
+use WCPay\PaymentMethods\Configs\Interfaces\PaymentMethodDefinitionInterface;
+use WCPay\PaymentMethods\Configs\Constants\PaymentMethodCapability;
+use WCPay\Constants\Currency_Code;
+use WCPay\PaymentMethods\Configs\Utils\PaymentMethodUtils;
+
+/**
+ * Class implementing the Link payment method definition.
+ */
+class LinkDefinition implements PaymentMethodDefinitionInterface {
+
+	/**
+	 * Get the internal ID for the payment method
+	 *
+	 * @return string
+	 */
+	public static function get_id(): string {
+		return 'link';
+	}
+
+	/**
+	 * Get the keywords for the payment method. These are used by the duplicate detection service.
+	 *
+	 * @return string[]
+	 */
+	public static function get_keywords(): array {
+		return [ 'link', 'stripe link' ];
+	}
+
+	/**
+	 * Get the Stripe payment method ID
+	 *
+	 * @return string
+	 */
+	public static function get_stripe_id(): string {
+		return PaymentMethodUtils::get_stripe_id( self::get_id() );
+	}
+
+	/**
+	 * Get the customer-facing title of the payment method
+	 *
+	 * @param string|null $account_country Optional. The merchant's account country.
+	 *
+	 * @return string
+	 */
+	public static function get_title( ?string $account_country = null ): string {
+		return __( 'Link', 'woocommerce-payments' );
+	}
+
+	/**
+	 * Get the title of the payment method for the settings page.
+	 *
+	 * @param string|null $account_country Optional. The merchant's account country.
+	 *
+	 * @return string
+	 */
+	public static function get_settings_label( ?string $account_country = null ): string {
+		return self::get_title( $account_country );
+	}
+
+	/**
+	 * Get the customer-facing description of the payment method
+	 *
+	 * @param string|null $account_country Optional. The merchant's account country.
+	 *
+	 * @return string
+	 */
+	public static function get_description( ?string $account_country = null ): string {
+		return __( 'Link autofills your customers\' payment and shipping details to deliver an easy and seamless checkout experience.', 'woocommerce-payments' );
+	}
+
+	/**
+	 * Get the list of supported currencies
+	 *
+	 * @return string[] Array of currency codes
+	 */
+	public static function get_supported_currencies(): array {
+		return [];
```

Comment:
The previous implementation only accepted USD. But it looks like Stripe revised their docs, and Stripe Link supports all currencies that `card` supports.